Essentials/src/plugin.yml

```diff
@@ -118,7 +118,7 @@ commands:
     aliases: [edisposal,trash,etrash]
   eco:
     description: Manages the server economy.
-    usage: /<command> <give|take|set|reset> <player> <amount>
+    usage: /<command> [give|take|set|reset] [player] <amount>
```

Comment:
Original was correct, no need for change here. The angle brackets (by common convention) indicate *required* arguments, while square brackets indicate *optional* arguments. In this command, the first two arguments are required, however the third can be optional. So you actually want to flip this here.